Corale.Colore/Core/IKeyboard.cs

```diff
@@ -81,7 +81,7 @@ public interface IKeyboard : IDevice
         /// <param name="first">Color to start from.</param>
         /// <param name="second">Color to reach, before going back to <paramref name="first" />.</param>
         [PublicAPI]
-        void Set(Color first, Color second);
+        void SetBreathingColors(Color first, Color second);
```

Comment:
Just `SetBreathing` for this, it's still the breathing effect. We can still have overloads for each effect, just not the extreme amount we had before.